server/src/main/java/org/apache/iotdb/db/exception/metadata/AliasAlreadyExistException.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.apache.iotdb.db.exception.metadata;
+
+import org.apache.iotdb.rpc.TSStatusCode;
+
+public class AliasAlreadyExistException extends MetadataException {
+
+  private static final long serialVersionUID = 7299770003548114589L;
+
+  public AliasAlreadyExistException(String path, String alias) {
+    super(String.format("Alias [%s] for Path [%s] already exist", alias, path),
+            TSStatusCode.PATH_ALREADY_EXIST_ERROR.getStatusCode());
```

Comment:
use the same error code as `PATH_ALREADY_EXIST_ERROR`?

Reply:
added a ALIAS_ALREADY_EXIST_ERROR